script.js

```diff
@@ -97,20 +85,16 @@ function renderRecipe() {
     recipeInnerModal.appendChild(link)
 
     recipeModal.style.display = 'flex'
-
 }
 
-
-
-
+getRecipeBtn.addEventListener("click", renderRecipe)
 
 //Get the recipe tags in an array
 function getCuisinesArray(recipes) {
     const cuisineArray = []
 
-    for (let recipe of recipes) {
-
-        for (let cuisine of recipe.recipeTags) {
+    for (const recipe of recipes) {
+        for (const cuisine of recipe.recipeTags) {
```

Comment:
We can use `const` for the variables used in a `for .. of` loop usually, as we're not going to assign new values to them.